### PR TITLE
Zcs-7731

### PR DESCRIPTION
**Pull request by akarambalkar**
I have created test plans for Businessplus, Standard and Professional editions for Zimbra X, please review the scripts and provide me a feedback if any changes are required.

### CODE REVIEW

**Comment by gordyt** [comment]
@akarambalkar You should probably rebase your PR branch and address the merge conflicts.

Also, please make sure all JMX files support the updated `users.csv` syntax, which is:

    <email-address-1>,<password>,<email-address-2>

File: tests/BSP/professional/zsoap/load.prop [hunk 1]
```diff
@@ -1,6 +1,17 @@
+<<<<<<< 074d4651e0b061fd59f12cfa700d4d1a3e8450b1
```
Comment (fatkudu) [inline tests/BSP/professional/zsoap/load.prop]: This looks wrong.  A bit like conflict markers?

File: tests/BSP/professional/zsoap/profiles/basic.prop [hunk 2]
```diff
@@ -1,6 +1,7 @@
 PROFILE.ZSOAP.type=sequence
 PROFILE.ZSOAP.sequence.1=AuthRequest(AUTHTYPE=user)
 PROFILE.ZSOAP.sequence.2=CreateFolderRequest
+<<<<<<< 074d4651e0b061fd59f12cfa700d4d1a3e8450b1
```
Comment (fatkudu) [inline tests/BSP/professional/zsoap/profiles/basic.prop]: Something bad going on here?

File: tests/BSP/standard/zsoap/profiles/basic.prop [hunk 3]
```diff
@@ -1,6 +1,7 @@
 PROFILE.ZSOAP.type=sequence
 PROFILE.ZSOAP.sequence.1=AuthRequest(AUTHTYPE=user)
 PROFILE.ZSOAP.sequence.2=CreateFolderRequest
+<<<<<<< 074d4651e0b061fd59f12cfa700d4d1a3e8450b1
```
Comment (fatkudu) [inline tests/BSP/standard/zsoap/profiles/basic.prop]: More strangeness?

File: tests/BSP/professional/zsoap/load.prop (hunk 1 above)
Comment (akarambalkar) [reply tests/BSP/professional/zsoap/load.prop]: Yes, I was trying to fix merge conflict and faced the above issues, I have fixed merge conflicts and committed to this branch again, please let me know for any issues.

**Comment by fatkudu** [comment]
There are a large number of files deleted by this change.  Is that intentional?

**Comment by fatkudu** [comment]
@akarambalkar I believe this pull request has been abandoned?  If so, please close it and perhaps delete the associated branch.

**Comment by akarambalkar** [comment]
I am closing this PR as there were merge conflict and rebase issues with the branch zcs-7731. I have created a new branch zcs-7816 and created a new Pull Request for the same. Hence I am closing this PR and will be deleting the branch zcs-7731 as there were lot of rebase issues with this branch.

**Pull request closed without merging**